person.rb

```diff
@@ -13,9 +45,22 @@ def can_use_services?
     of_age? || parent_permission
   end
 
+  def correct_name
+    @name
+  end
+
   private
 
   def of_age?
     @age >= 18
   end
 end
+
+person = Person.new(22, 'maximilianus')
+puts person.correct_name
+
+capitalized_person = CapitalizeDecorator.new(person)
+puts capitalized_person.correct_name
+
+capitalized_trimmed_person = TrimmerDecorator.new(capitalizedPerson)
+puts capitalized_trimmed_person.correct_name
```

Comment:
Great design @J-C-S-V 🤝

- [ ] I would suggest using `capitalized_trimmed_person = TrimmerDecorator.new(capitalized_person)` instead of `capitalized_trimmed_person = TrimmerDecorator.new(capitalizedPerson)` for you to be able to trim a class.

Reply:
Hello @1GORDON,

Thanks for the heads up!